tests/test_wave_03.py

```diff
@@ -54,13 +54,16 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert INTRIGUE_3 in friends_unique_movies
+    assert friends_unique_movies.count(INTRIGUE_3) == 1
```

Comment:
Interesting, checking for both that `INTRIGUE_3` ends up in the right place. Is it necessary to check that it's only in the `friends_unique_movies` list once?